src/aioswitcher/bridge.py

```diff
@@ -268,14 +278,14 @@ def is_switcher_originator(self) -> bool:
             or len(self.message) == 159  # Switcher Runner and RunnerMini
         )
 
-    def get_ip(self) -> str:
-        """Extract the IP address from the broadcast message."""
+    def get_ip_type1(self) -> str:
```

Comment:
going back to my common theme, can we do better than type1 vs type2?
we can even inject the get IP function based on the device type to make it more clear, no?

Reply:
Same as above....